pymbolic/primitives.py

```diff
@@ -1551,7 +1572,12 @@ def get_extra_properties(self):
 
 @expr_dataclass()
 class Substitution(Expression):
-    """Work-alike of sympy's Subs."""
+    """Work-alike of sympy's Subs.
+
+    .. autoattribute:: child
+    .. autoattribute:: variables
+    .. autoattribute:: values
+    """
```

Comment:
I don't think these are in the docs. Worth adding?

Reply:
Done.